### DISCUSSION TITLE
When there is an error or invalid, scrollToInvalid throwing an error ??

### DISCUSSION THREAD

**Discussion by prajwalbharadwaj** [vueform/vueform#354] [Questions]
scrollToInvalid doesnt seems to be working and throwing this error. 

How to scroll to invalid element?

<img width="651" alt="image" src="https://github.com/user-attachments/assets/67137229-1bb4-4b5a-bd42-920c67d37b4b">

**Comment by adamberecz** [comment]
Please update to the latest version, as this got fixed recently.

**Reply by prajwalbharadwaj** [reply]
@adamberecz Can you please let me know in which version it is fixed ??

We are using `1.10.10` which is the latest version

**Comment by digino** [comment]
@adamberecz
I'm not seeing the error after update to the last version(1.11.0). But it seems like not working. I build a multistep form of more than 100 fields. When a field is required and i click the next button, the next button is disabled but not scrolled to the invalid field.

**Reply by prajwalbharadwaj** [reply]
Totally agreed. It doesn't scroll when there is an issue
